src/Ledger/Utxo/Properties.lagda

```diff
@@ -39,23 +40,35 @@ open Tx
 open Equivalence
 open Properties
 
-import Data.Nat.Tactic.RingSolver as ℕ
```

Comment:
This import seems unnecessary.  Also, what happens when we give both `Data.Nat` and `Data.Nat.Tactic.RingSolver` the same handle?  That's frightening. 😨

Reply:
reverted ∵ change requests